CONTRIBUTING.md

```diff
@@ -93,3 +93,35 @@ For merging, you should:
        Travis to run the tests for each change you add in the pull request.
        Because testing here will delay tests by other developers, please ensure
        that the code passes all tests on your local system first.
+
+Project Style Guidelines
+------------------------
+
+In general, pysat follows PEP8 and numpydoc guidelines.  Pytest runs the unit
+ and integration tests, flake8 checks for style, and sphinx-build performs
+documentation tests.  However, there are certain additional style elements that
+have been settled on to ensure the project maintains a consistent coding style.
+These include:
+
+* Line breaks should occur before a binary operator (ignoring flake8 W503)
+* Combine long strings using `join`
+* Preferably break long lines on open parentheses rather than using `\`
+* Avoid using Instrument class key attribute names as unrelated variable names:
+  * `platform`
+  * `name`
+  * `tag`
+  * `inst_id`
+* The pysat logger is imported into each sub-module and provides status updates
+  at the info and warning levels (as appropriate)
+* Several dependent packages have common nicknames, including:
+  * `import datetime as dt`
+  * `import numpy as np`
+  * `import pandas as pds`
+  * `import xarray as xr`
+* All classes should have `__repr__` and `__str__` functions
+* Docstrings use `Note` instead of `Notes`
```

Comment:
Should we note the general format of the docstrings is numpydoc except for... (correct values here)

Reply:
Well, of course. I did read things ahead of time but clearly not well enough.